### PR TITLE
Add multitenancy support to console

### LINKED ISSUE #37: Support multitenancy for console
Right now the console connects to the address-controller using the service environment. These environment variables are not available when they run in separate namespaces. 

Work has already been done to prepare the console to use API_SERVER_HOSTNAME environment if available. The remaining work is to expose this environment variable so that it will be used by the console.

### PR DESCRIPTION
**Pull request by lulf**
This fixes #37

### CODE REVIEW

File: include/admin.jsonnet [hunk 1]
```diff
@@ -85,7 +85,16 @@ local common = import "common.jsonnet";
                       {
                         "name": "CONFIGURATION_SERVICE_PORT",
                         "value": "5672"
-                      }]),
+                      },
+                      {
+                        "name": "API_SERVER_HOSTNAME",
```
Comment (grs) [inline include/admin.jsonnet]: Minor suggestion on naming... perhaps ADDRESS_CONTROLLER_HOST would be more obvious? Or else ADDRESS_SERVICE_HOST (to emulate loosely the kubernetes naming convention)?

Comment (lulf) [inline include/admin.jsonnet]: So my reason for using this name is that the API  would eventually be part of the enmasse API server which would be a 3rd party API server in kubernetes. So the name is following the kubernetes convention in that sense.

Comment (lulf) [inline include/admin.jsonnet]: So I think ADDRESS_CONTROLLER_HOST is wrong in that sense. ADDRESS_SERVICE_HOST is better, but it's not only about addresses anymore... But i agree _HOST is better than _HOSTNAME

Comment (grs) [inline include/admin.jsonnet]: Fair points, and it is only a name so not a huge issue at this point. API_SERVER is a little too ambiguous/generic for my taste (though as an env var I can certainly live with it :). What about MESSAGING_CONTROL or MESSAGING_ADMIN or something a little more descriptive of its actual function?

Comment (lulf) [inline include/admin.jsonnet]: MESSAGING_ADMIN_API_HOST  ?

Comment (grs) [inline include/admin.jsonnet]: I prefer MESSAGING_ADMIN_SERVICE_HOST, but I would be fine with your suggestion .

**Pull request merged**